app/database.py

```diff
@@ -11,7 +11,7 @@ def __init__(cls, db_url):
     @classmethod
     def init_db(cls, db_url):
         try:
-            cls.connection = psycopg2.connect(db_url)
+            cls.connection = psycopg2.connect(user="postgres",password="postgres",host="127.0.0.1",port="5432",database='zaobora_database')
```

Comment:
Please if your code is breaking I can help you resolve it. Changing lines like this is jot ideal

Reply:
Code working fine locally. Would like help better code commit practices.